README.md

```diff
@@ -46,4 +46,4 @@ A massive thank you goes to these wonderful people who have contributed to and b
 
 <!-- ALL-CONTRIBUTORS-LIST:END -->
 
-This project follows the [all-contributors](https://github.com/all-contributors/all-contributors) specification. Please read our [Guidelines](.contributing) on how to contribute to this Glossary. The Glossary is maintained and built by our community so if you believe a term is missing, please feel welcome to add or suggest content. For information on how to do this, read through the [Contributing](./contributing.md) information.
+This project follows the [all-contributors](https://github.com/all-contributors/all-contributors) specification. Please read our [Guidelines](docs/contributing.md) on how to contribute to this Glossary. The Glossary is maintained and built by our community so if you believe a term is missing, please feel welcome to add or suggest content. For information on how to do this, read through the [Contributing](docs/contributing.md) information.
```

Comment:
```suggestion
This project follows the [all-contributors](https://github.com/all-contributors/all-contributors) specification. Please read our [Contributors Guidelines](docs/contributing.md) on how to contribute to this Glossary. The Glossary is maintained and built by our community so if you believe a term is missing, please feel welcome to add or suggest content.
```

Reply:
I thought it'd be nice to keep the "Contributors Guidelines" file name consistent and I removed some content that seemed to me like repetition.
Looking at the files, I can see that the path is correct but is there a way to clink on the link while it's in a PR and make sure it takes you to the right place?

Reply:
......it worked locally 🤣 

Okay being serious though I tend to go on the repo on that branch and then click through to check that it worked (that was how I figured out they were broken to begin with)